src/rules/preferForOfRule.ts

```diff
@@ -47,17 +47,15 @@ interface IIncrementorState {
 }
 
 // a map of incrementors and whether or not they are only used to index into an array reference in the for loop
-interface IncrementorMap {
-    [name: string]: IIncrementorState;
-}
+type IncrementorMap = Map<string, IIncrementorState>;
 
 class PreferForOfWalker extends Lint.BlockScopeAwareRuleWalker<{}, IncrementorMap> {
     public createScope() {
-        return {};
+        return new Map();
```

Comment:
this method could just return `void` (need to update type parameters). `scope` is never used anywhere, only `blockScope` is needed.